Refine grammar and style in the documentation's language section. (#39)
tact-lang/tact-docs: pages/language/guides/changelog.mdx

- `String` literals and variables
- `Int.toString()` and `Int.toFloatString()`
- `StringBuilder` for gas-efficient string building
- Global compile-time `ton` function that converts a string to `Int` during compile time.
- `checkDataSignature` similar to func `check_data_signature`
- `String.asComment` for conversion text to a comment payload

> - `Resumable` trait, allows to resumption of contract operations once it was stopped

Reviewer: ```suggestion
- `Resumable` trait, allows to resume operations, once the contract was stopped
```
